server/src/api/donations/donations.service.ts

```diff
@@ -52,19 +52,20 @@ export const update = async (r: any): Promise<response> => {
     text: `
       UPDATE donations
       SET (
-        donorid,
+        customerid_fk,
```

Comment:
Should be `contactid_fk`

Reply:
This one isn't actually linked to the contact table

Reply:
Should it be?

Reply:
It is in [task_sql](https://github.com/WonderTix/WonderTix/tree/development/task_sql)/mywtix_schema.PNG

Reply:
Oh I just saw that edit. I think the most up to date schema is the one I linked. We'll have to update it again though with the renamed tables

Reply:
Gotcha. should I just swap it and push it broken, so when the change is merged it will work?

Reply:
That change already got merged so the table is already named `contacts`

Reply:
You might need to reset your postgres docker container for the changes to work

Reply:
Ooooh gotcha.